src/shared/VacationTable/VacationTable.tsx

```diff
@@ -149,32 +177,63 @@ const VacationTable: FC = () => {
               <TableCell sx={{ width: "150" }} align="center">
                 Type
               </TableCell>
-              <TableCell align="right">Start date</TableCell>
-              <TableCell align="right">End date</TableCell>
+              <TableCell align="center">Start date</TableCell>
+              <TableCell align="center">End date</TableCell>
               <TableCell sx={{ width: "500px" }} align="center">
                 Notes
               </TableCell>
               <TableCell align="center">Action</TableCell>
             </TableRow>
           </TableHead>
-          {rows.length < 1 ? null : (
+          {data.length < 1 ? null : (
             <TableBody>
               {(rowsPerPage > 0
-                ? rows.slice(
+                ? data.slice(
                     page * rowsPerPage,
                     page * rowsPerPage + rowsPerPage
                   )
-                : rows
-              ).map((row: any) => (
-                <TableRow key={row.name}>
+                : data
+              ).map((vacationInfo: IDataVacation) => (
+                <TableRow key={vacationInfo.id}>
                   <TableCell component="th" scope="row">
-                    {row.id}
+                    {vacationInfo.id.slice(0, 4)}
+                  </TableCell>
+                  <TableCell align="center">{vacationInfo.type}</TableCell>
+                  <TableCell align="center">
+                    {typeof vacationInfo.startDate === "string"
```

Comment:
why do you need this check?
I assume that you might have a data type misled. If so, that means that you should convert it to dayjs format earlier and just display here.